### DISCUSSION TITLE
HUGE IGNORANT ALERT. Please, remove this thing. Make it smaller, normal, or send this through email.

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#19072] [Other Feature Feedback, Questions, & Ideas]
This thing is annoying as fuck, for god's sake.
![image](https://user-images.githubusercontent.com/1277920/174874131-720a64a4-ec5f-4464-a4be-90ae8b352012.png)

**Comment by leoheck** [comment]
After seeing this I don't even want to use Github anymore. How do I at least remove this thing for myself?

**Comment by NeuralAIM** [comment]
Use **adblock**, where you can block any element separately. And then you won't see this ad anymore :)

**Comment by leoheck** [comment]
Good workaround. But this huge banner feels like Microsoft doing stupid things as usual.

**Comment by kevin-he-01** [comment]
Exactly! Micro$oft is trying to push ads to everyone that they cannot close without using an ad blocker. This is what I get because I was in the copilot beta (and I don't like to pay them):
<img width="642" alt="image" src="https://user-images.githubusercontent.com/32375681/180077438-2f67b172-5043-4300-b0f1-c1fea79d3186.png">

See? No x button unlike some other banners and this is totally intentional. There is no way I can close it without paying $$$$.

Even if you try to block it with an ad blocker you will block the real warnings (Ex. we noticed suspicious activity on your account) as well. What a wonderful business strategy to put a crying wolf alert! This is There is no way to distinguish this warning from other ones.
